v1.3.0 - Upgrade autoscaling api group version

ralongit opened the pull request:
- Added a condition to the horizontal pod autoscaler to upgrade the api group version if the Kubernetes version is 1.23+ as the previous one was deprecated in newer versions.
- Updated logzio-monitoring chart dependency version.
- Upgraded chart versions to logzio-monitoring & logzio-telemetry charts

ralongit committed:
v1.2.1 - Upgrade autoscaling api group version

- Added a condition to the horizontal pod autoscaler to upgrade the api group version if the Kubernetes version is 1.23+ as the previous one was deprecated in newer versions.
- Updated logzio-monitoring chart dependency version.
- Upgraded chart versions to logzio-monitoring & logzio-telemetry charts

ralongit committed:
Update logzio-monitoring chart version to 1.7.1

ralongit committed:
Change capabilities syntax to Helm3 & revert api version to betav1

mirii1994 reviewed (changes requested):
Great, see my comment :)

ralongit committed:
Bump logzio-k8s-telemetry to version 1.3.0

- Updated readme files version
- Updated charts versions

ralongit committed:
Remove replicasCount from daemonset

- Daemonsets do not include the field "replicas" in their spec, removes missing replicas warning.
closes #323
- Autoscaling commit solves #340

ralongit committed:
Bump logzio-monitoring chart version to 1.8.0

mirii1994 reviewed (approved):
Well done!
